### PR TITLE
[Umb8] Lee's code review patch

### PR DESCRIPTION
**Pull request by leekelleher**
Here's all my patches following up on my code review for PR #5.

### CODE REVIEW

File: src/Our.Umbraco.RobotsTxtEditor/Controllers/RobotsTxtEditorApiController.cs [hunk 1]
```diff
@@ -18,9 +17,9 @@ public class RobotsTxtEditorApiController : UmbracoAuthorizedApiController
         [HttpGet]
         public RobotsTxtEditorModel GetRobotsText()
         {
-            var filePath = HttpContext.Current.Server.MapPath("~/robots.txt");
+            var filePath = IOHelper.MapPath("~/robots.txt");
```
Comment (MMasey) [inline src/Our.Umbraco.RobotsTxtEditor/Controllers/RobotsTxtEditorApiController.cs:20]: I didn't know about IOHelper so thanks 😄

**Comment by MMasey** [comment]
The changes all look good and make sense to me Lee 👍 Thank you 😄

File: src/Our.Umbraco.RobotsTxtEditor/Controllers/RobotsTxtEditorApiController.cs (hunk 1 above)
Comment (LottePitcher) [reply src/Our.Umbraco.RobotsTxtEditor/Controllers/RobotsTxtEditorApiController.cs:20]: ^^ what he said!

**Review by LottePitcher — COMMENTED** [review]
oh that's what 'gitkeep' is for! I always use 'dummy.txt' but think i'm gonna change to .gitkeep from now on. nice!

You gonna accept the PR @MMasey ?

**Comment by leekelleher** [comment]
🎉 PR reviewed and merged in, nice one! 🎉 

@LottePitcher re: `.gitkeep`, I was the same, then noticed other people/repos using that convention too. 😄

**Pull request merged**